From 95168e8f16fc7a9ea6493789e0c95d8f061010b5 Mon Sep 17 00:00:00 2001
From: Chanh Nguyen <chanh@os.amperecomputing.com>
Date: Mon, 27 Mar 2023 16:20:26 +0700
Subject: [PATCH 5/7] adcsensor: Disable io_uring build

When the adcsensor uses the io_uring lib,
That causes a reading error to LTC2497 device (an ADC/I2C
device).

This is workaround for temp solution. That will be reverted after we have
a better solution with the ltc2497 driver.

Signed-off-by: Chanh Nguyen <chanh@os.amperecomputing.com>
---
 src/meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/meson.build b/src/meson.build
index 5eeda20..d0aed71 100644
--- a/src/meson.build
+++ b/src/meson.build
@@ -82,7 +82,6 @@ if get_option('adc').enabled()
             thresholds_dep,
             utils_dep,
         ],
-        cpp_args: uring_args,
         install: true,
     )
 endif
-- 
2.34.1

